OpenRA.Mods.Common/Traits/Buildings/GivesBuildableArea.cs

```diff
@@ -0,0 +1,35 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Tag trait for actors with `AreaTypes`.")]
+	public class GivesBuildableAreaInfo : ConditionalTraitInfo
+	{
+		[Desc("Accepted `AreaTypes` types. Leave empty to accept all types.")]
```

Comment:
`to accept` doesn't sound right, since this trait `gives`, it does not `accept`. ;)
Also at a glance it doesn't seem like leaving this empty will grant you any buildable area at all.

Reply:
It indeed doesn't. It should be complately removed.